Academia-750/academia750-frontend-vue
# DBT-759 Feat/student materials recordings

## Context


In this task we expect to have to see the students materials and recordings 


[Task Link](https://www.notion.so/tianlu/Frontend-Table-listing-the-material-that-can-be-filtered-by-content-label-or-lesson-e0a18b3baeb94afab13d1b7a1efe3980?pvs=4)

## Test



- Login as a student
- check the sidebar and see if you can find the materials and recordings tabs
- click on them and test them



## Platform



## Proof of Concept


<img width="1010" alt="image" src="https://github.com/Academia-750/academia750-frontend-vue/assets/79788219/a9bd534e-0f9e-49c0-9d5e-cadfeff7d322">
